src/main/scala/io/iohk/ethereum/network/package.scala

```diff
@@ -19,4 +25,26 @@ package object network {
       Hex.decode(nodeId)
     curve.getCurve.decodePoint(bytes)
   }
+
+  def loadAsymmetricCipherKeyPair(filePath: String): AsymmetricCipherKeyPair = {
```

Comment:
maybe I am missing something but this method name is `loadAsymmetricCipherKeyPair` but this method writes keys to file `file.write(keyFileFormat.format(pub, priv))` should we change the name?